### PR TITLE
Enable CoreFx tests

### PR DESCRIPTION
**Pull request by NikolaMilosavljevic**

### CODE REVIEW

File: build.proj [hunk 1]
```diff
@@ -13,6 +13,13 @@
     <MSBuild Projects="repos\$(RootRepo).proj" Targets="Build" BuildInParallel="$(BuildInParallel)" StopOnFirstFailure="true" />
   </Target>
 
+  <Target Name="RunTests" DependsOnTargets="">
```
Comment (crummel) [inline build.proj]: Should this be blank or is this targets this depends on?

File: repos/known-good-tests.proj [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+  <PropertyGroup>
+    <RepoApiImplemented>false</RepoApiImplemented>
+    <OrchestratedManifestBuildName>N/A</OrchestratedManifestBuildName>
+    <SkipEnsurePackagesCreated>true</SkipEnsurePackagesCreated>
+  </PropertyGroup>
+
+  <!-- This project file serves a couple of purpose.
```
Comment (crummel) [inline repos/known-good-tests.proj]: Comment is inaccurate for test projects.

**Comment by crummel** [comment]
It looks like the CI failures are something to do with Target RID changes, but I'm not seeing anything that should affect that.  Were you seeing anything like that locally?

**Comment by NikolaMilosavljevic** [comment]
Abandoning due to CI issues to merge new changes into fork and test it locally.

**Pull request closed without merging**